main.R

```diff
@@ -0,0 +1,28 @@
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+#Main directory should be: ~/ds-pipelines-targets-1
+
+#1_fetch----
+setwd('./1_fetch/src')
+source('getData.R')
+data = getData(outDir = '../out')
+
+#2_process----
+setwd('../../2_process/src/')
+source('processData.R')
+# Prepare the data for plotting and save to file
+data = processData(data,
+                   outDir = '../out/')
+
+RMSEs = writeSummary(data, 
+                     outDir = '../out/')
+
+#3_visualize----
+setwd('../../3_visualize/src/')
```

Comment:
This all looks great. We'll get away from `setwd()` in the next few PRs, but it makes sense for now. I like that you colocated the `source()` calls with the function calls that use those scripts.